### ISSUE TITLE
ReLU

### ISSUE THREAD

**Issue by xiaoqiang765** [developer0hye/SKNet-PyTorch#6]
sorry, i was wondering why using 'inplace=False' instead of 'inplace=True' in some ReLU;
After i implement your code, i find if input shape [1, x, 1, 1],batchnorm error, i wondering why using conv in sknet instead of linear in senet, look forward your reply, thanks

**Comment by developer0hye** [comment]
@wuyongqiang765 
You don't need to sorry to me. You pointed important part to save memory usage. I changed `inplace=False' into `inplace=True`.
